mteb/abstasks/AbsTaskRetrieval.py

```diff
@@ -287,6 +287,8 @@ def load_data(self, **kwargs):
                     "trust_remote_code", False
                 ),
             ).load(split=split)
+            
+            
```

Comment:
No op. Can we remove this change? Same for the new line below.

Reply:
removed.